### PR TITLE
Correct release note on frozen to_s in 2.7.0-preview3 (de, es, ko, zh_cn, zh_tw)

### PR DESCRIPTION
**Pull request by aleksandrilyin**
Hello!
Done as in PR #2308 for de, es, ko, zh_cn and zh_tw translations.

### CODE REVIEW

**Comment by Quintus** [comment]
Looks good for me (de).

File: zh_cn/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ end
 
   * `--jit-max-cache` 的默认值从 1,000 调整到 100。
 
-* Symbol#to_s`, `Module#name`, `true.to_s`, `false.to_s` 和 `nil.to_s` 现在始终返回一个冻结（frozen）字符串。返回的字符串始终和给定的对象相等。 [实验性]  [[功能 #16150]](https://bugs.ruby-lang.org/issues/16150)
+* ~~`Symbol#to_s`~~ (已回复), `Module#name`, `true.to_s`, `false.to_s` 和 `nil.to_s` 现在始终返回一个冻结（frozen）字符串。返回的字符串始终和给定的对象相等。 [实验性]  [[功能 #16150]](https://bugs.ruby-lang.org/issues/16150)
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]:
```suggestion
* ~~`Symbol#to_s`~~ (已撤回), `Module#name`, `true.to_s`, `false.to_s` 和 `nil.to_s` 现在始终返回一个冻结（frozen）字符串。返回的字符串始终和给定的对象相等。 [实验性]  [[功能 #16150]](https://bugs.ruby-lang.org/issues/16150)
```

cc @dsh0416

File: zh_tw/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ end
 
   * `--jit-max-cache` 的預設值從 1,000 調整至 100。
 
-* ~~`Symbol#to_s`, `Module#name`, `true.to_s`, `false.to_s` 和 `nil.to_s` 現在始終返回一個凍結（frozen）字串。返回的字串始終和給定的物件相等。 [實驗性質] [[Feature #16150]](https://bugs.ruby-lang.org/issues/16150)~~ 已回復
+* ~~`Symbol#to_s`~~ (已回復), `Module#name`, `true.to_s`, `false.to_s` 和 `nil.to_s` 現在始終返回一個凍結（frozen）字串。返回的字串始終和給定的物件相等。 [實驗性質] [[Feature #16150]](https://bugs.ruby-lang.org/issues/16150)
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]:
```suggestion
* ~~`Symbol#to_s`~~ (已取消), `Module#name`, `true.to_s`, `false.to_s` 和 `nil.to_s` 現在始終返回一個凍結（frozen）字串。返回的字串始終和給定的物件相等。 [實驗性質] [[Feature #16150]](https://bugs.ruby-lang.org/issues/16150)
```

**Comment by aleksandrilyin** [comment]
Thanks @JuanitoFatas for improvements and @marocchino for approval. Committed.

**Comment by aleksandrilyin** [comment]
Hi!
/cc @dsh0416 please review these changes.

**Comment by aleksandrilyin** [comment]
/cc @JuanitoFatas
@JuanitoFatas can you merge these changes please if all good?

**Comment by JuanitoFatas** [comment]
@aleksandrilyin awesome, thank you so much!!! ❤️

**Pull request merged**